Common/src/main/java/com/tabnineCommon/chat/commandHandlers/context/workspace/FindSymbolsCommandExecutor.kt

```diff
@@ -2,11 +2,28 @@ package com.tabnineCommon.chat.commandHandlers.context.workspace
 
 import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
+import com.tabnineCommon.chat.commandHandlers.utils.StringCaseConverter
 import com.tabnineCommon.chat.commandHandlers.utils.SymbolsResolver
+import java.util.stream.Stream
+import kotlin.streams.toList
+
+private const val MAX_RESULTS_PER_SYMBOL = 5
 
 class FindSymbolsCommandExecutor : CommandsExecutor {
     override fun execute(arg: String, editor: Editor, project: Project): List<String> {
-        return SymbolsResolver.resolveSymbols(project, editor.document, arg, 5)
+        val camelCaseArg = StringCaseConverter.toCamelCase(arg)
+        val snakeCaseArg = StringCaseConverter.toSnakeCase(arg)
+        val camelCaseSymbols = SymbolsResolver.resolveSymbols(
+            project, editor.document, camelCaseArg,
+            MAX_RESULTS_PER_SYMBOL
+        )
+        val snakeCaseSymbols = SymbolsResolver.resolveSymbols(
```

Comment:
is this case sensitive?

Reply:
yes:
![image](https://github.com/codota/tabnine-intellij/assets/67855609/faba9ddb-d489-4f70-840a-97589ca08264)
![image](https://github.com/codota/tabnine-intellij/assets/67855609/0f8ee458-c242-47f3-a504-c72abce0fe9c)